src/applications/find-forms/widgets/createFindVaFormsPDFDownloadHelper/DownloadPDFGuidance.js

```diff
@@ -0,0 +1,65 @@
+// Dependencies.
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+// relative imports
+import DownloadPDFModal from './DownloadPDFModal';
+import InvalidFormDownload from './InvalidFormAlert';
+import { onDownloadLinkClick, sentryLogger } from './index';
+import { showPDFModal } from '../../helpers/selectors';
+
+// DownloadPDFGuidance sole purpose is to render the react widget depending on params (PDF is valid or not)
+const DownloadPDFGuidance = ({
+  downloadUrl,
```

Comment:
Can we add prop types? 🙂

Reply:
I didn't do it here since it is just a function and not an actual React Component.